File: scratch/test_smtp.py
import smtplib
import os

try:
    server = smtplib.SMTP("smtp.gmail.com", 587)
    server.starttls()
    server.login("thelive.inbuddy@gmail.com", "umixdalimqsvdjas")
    print("SUCCESS: Logged in to SMTP successfully!")
    server.quit()
except Exception as e:
    print("FAILED:", str(e))

File: scratch/test_smtp.js
const nodemailer = require("nodemailer");

async function testSmtp() {
  try {
    let transporter = nodemailer.createTransport({
      host: "smtp.gmail.com",
      port: 587,
      secure: false, // true for 465, false for other ports
      auth: {
        user: "thelive.inbuddy@gmail.com",
        pass: "umixdalimqsvdjas",
      },
    });

    let info = await transporter.verify();
    console.log("SUCCESS: Logged in to SMTP successfully!");
  } catch (e) {
    console.log("FAILED:", e.message);
  }
}

testSmtp();
